### DISCUSSION TITLE
Disable "Row added"

### DISCUSSION THREAD

**Discussion by SkylerLutz** [IgniteUI/igniteui-angular#11252] [Q&A]
## Question  
Hi -- when I add a row to my grid, I get a message in the snackbar that says "Row added". How can I prevent this message from appearing entirely? Thanks. If it's not possible to disable, how can I customize the text? Thanks

**Comment by mddragnev** [comment]
Hello, @SkylerLutz
Currently, we do not expose templating of the add row snackbar. You can only customize the text of the snackbar and the text of the snackbar action. You can do so from the grid `resourceStrings` property. You can bind to the `AfterViewInit` hook and change them from there. It should look something like this:
```
ngAfterViewInit(): void {
        this.grid.resourceStrings.igx_grid_snackbar_addrow_label = 'Adding row';
        this.grid.resourceStrings.igx_grid_snackbar_addrow_actiontext = 'Find row'
    }
```

Please, let me know if you have any other questions.
